ui/src/components/yaml/styles.module.less

```diff
@@ -223,7 +221,9 @@
       justify-content: space-between;
```

Comment:
Introducing a linear gradient may not be intended and could cause unexpected visual effects.